src/components/common/UrlFiltersProvider/UrlFiltersProvider.test.tsx

```diff
@@ -129,13 +129,13 @@ describe('useUrlFilters', () => {
     });
   });
 
-  it('should clear the search term', async () => {
+  it('should clear the filters and search term', async () => {
     const { result } = renderHook(() => useUrlFilters(), {
       wrapper: Wrapper,
     });
 
     act(() => {
-      result.current.clearSearchTerm();
+      result.current.clearFilters();
     });
 
     expect(result.current.searchTerm).toBe('');
```

Comment:
Sorry, I missed this before. But we should also test that the filters are reset.

```suggestion
    expect(result.current.activeFilters).toEqual({});
    expect(result.current.searchTerm).toBe('');
```